implementations/rust/ockam/ockam_command/src/pager.rs

```diff
@@ -36,8 +36,17 @@ fn paginate_with(pager_binary_path: PathBuf, help: clap::Error) -> miette::Resul
             .stdin
             .take()
             .expect("Failed to get pager's stdin");
+        // Check if the output is a terminal
+        let rendered_text = if atty::is(Stream::Stdout) {
```

Comment:
You can do the same without adding a new dependency:

```rust
if Term::stdout().is_term() {
    ...
}
```

Reply:
@adrianbenavides Hi, thank you for your feedback! I updated my code accordingly!